src/Kernel32.Shared/NTStatusException.cs

```
@@ -89,7 +92,7 @@ private static string GetMessage(NTSTATUS status)
             string insert = $"NT_STATUS {GetSeverityString(status)}: {statusAsString}";
             string message = null;
 #if DESKTOP
-            message = status.GetMessage();
+            message = status.GetMessage(0x409);
```

Shouldn't we propagate `dwLanguageId` instead of a hard-coded `0x409` here?

---

this is outdated now.